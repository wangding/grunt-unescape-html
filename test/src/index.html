<!DOCTYPE html>
<html lang="zh">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Document</title>
</head>
<body>
  <h1>&#x4e2d;&#x56fd;</h1>
  <h1>&#x4e2d;&nbsp;&#x56fd;</h1>
</body>
</html>
