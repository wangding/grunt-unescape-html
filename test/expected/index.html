<!DOCTYPE html>
<html lang="zh">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Document</title>
</head>
<body>
  <h1>中国</h1>
  <h1>中&nbsp;国</h1>
</body>
</html>
